webapp/lib/feeds.py

```diff
@@ -59,6 +59,10 @@ def get_rss_feed_content(url, offset=0, limit=None):
         )
         content = []  # Empty response
 
+    if exclude_items_in:
+        exclude_ids = [x['guid'] for x in exclude_items_in]
+        content = filter(lambda x: x['guid'] not in exclude_ids, content)
```

Comment:
I am going to leave this as I haven't written it... but I agree.... @deadlight  to fix please later